source/libs/tokenizer.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This custom tokenizer for remark-parse is needed so that
+ * hashtags (#refinedtwitter) and mentions (@sindresorhus) are
+ * not sanitized when the markdown is compiled to HTML.
+ */
+
+function mentions(eat, value, silent) {
+	const match = /^@(\w+)/.exec(value);
+
+	if (match) {
```

Comment:
Can you destructure `match` here and give the parts names to make it more readable?

Reply:
Not sure what you mean

Reply:
```jsif (match) {

Reply:
```js

Reply:
```js
if (match) {
  const [foo, bar] = match;
```

With `foo` and `bar` having relevant names.

Reply:
I don't think this change will add much. `match[0]` is always something like `#sindresorhus` or `@sindresorhus` and `match[1]` is always the string with the `#` or `@` removed (`sindresorhus`).

Reply:
Well I at least can't think of good names for these.